.travis.yml

```diff
@@ -39,15 +39,21 @@ env:
 
 before_install:
   - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - npm install -g markdown-link-check
 
 script:
 - set -o pipefail
 
+- if [ $CHECK_MARKDOWN == "YES" ]; then
+      echo -en 'travis_fold:start:script.1\\r';
```

Comment:
actually, it's probably ok *not* to fold this. it's only going to run on the first build, right?

doesn't travis fold most steps automatically after completion?